docs/customization/animations.ipynb

```diff
@@ -0,0 +1,215 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0",
+   "metadata": {},
+   "source": [
+    "# Animations\n",
+    "\n",
+    "To create animations with Plopp, we make use of the [node mechanism](custom-interfaces.ipynb)\n",
+    "and leverage [Matplotlib's support for animations](https://matplotlib.org/stable/users/explain/animations/animations.html).\n",
+    "\n",
+    "## 1D animations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import plopp as pp\n",
+    "from plopp.data.examples import three_bands\n",
+    "import matplotlib.animation as animation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2",
+   "metadata": {},
+   "source": [
+    "We load the data and display it in a figure:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = three_bands()\n",
+    "da.plot(norm=\"log\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4",
+   "metadata": {},
+   "source": [
+    "We now create an animation where we are scanning the vertical `y` direction of the data,\n",
+    "making a plot of each slice in each step of the animation.\n",
+    "\n",
+    "On every call to the `update` function passed to Matplotlib's `FuncAnimation`,\n",
+    "we update the index node and notify the children nodes of the change.\n",
+    "This updates the figure at every step.\n",
+    "\n",
+    "Finally, we save the animation to a `.gif` file.\n",
```

Comment:
Can you also show how to display the animation in a notebook directly? Or is this not possible with mpl?